syntax = "proto3";

package dataservice;

service DataService {
  rpc SendData (DataRequest) returns (DataResponse) {}
}

message DataRequest {
  string data = 1;
}

message DataResponse {
  string message = 1;
}
